Src/BootCamp.Chapter/Utils.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace BootCamp.Chapter
+{
+    public static class Utils
+    {
+        public static int ConvertToInt(string input)
+        {
+            int.TryParse(input, out int value);
+            return value;
+        }
+
+        public static bool IsNumeric(string input)
+        {
+            return int.TryParse(input, out _);
+        }
```

Comment:
Looking at file UserInteraction you can see that you check for being numeric and then parse it again (ConvertToInt).
Do this
```csharp
string cipherKey = Console.ReadLine();
// bool isNumeric = int.TryParse(cipherKey, out int result); if you can't use result after the if
if (!int.TryParse(cipherKey, out int result))
{
    return;
}
var decryptedMessage = CaesarCipher.Decrypt(encryptedMessage, result);
```
instead of
```csharp
string cipherKey = Console.ReadLine();
if (!Utils.IsNumeric(cipherKey))
{
    return;
}
var decryptedMessage = CaesarCipher.Decrypt(encryptedMessage, Utils.ConvertToInt(cipherKey));
```